Dockerfile

```diff
@@ -2,11 +2,8 @@ FROM cccs/assemblyline-v4-service-base:latest AS base
 
 ENV SERVICE_PATH emlparser.emlparser.EmlParser
 
-USER root
-RUN apt-get update && apt-get install -y git && rm -rf /var/lib/apt/lists/*
```

Comment:
I understand getting rid of the installation of git, but why are you removing the `apt-get update`?

Reply:
Only needed apt-get update to install git, otherwise it's not really needed if I don't need to install any extract packages (other than updating any pre-existing from the base image).